.rubocop_todo.yml

```diff
@@ -50,6 +50,12 @@ Lint/UnusedMethodArgument:
 Metrics/AbcSize:
   Max: 82
 
+# Offense count: 1
```

Comment:
This was needed to make rubocop happy for the change in the PR.  Not sure about the other changes to the file.